main.js

```diff
@@ -0,0 +1,157 @@
+const apiKey = '465964c6411f5ad0764a0279bef46754';
+let currentWeather = [];
+let fiveDayWeather = [];
+
+document.querySelector('.search').addEventListener('click', function (){
+    let city = document.querySelector("#search-query").value
+    
+    document.querySelector('#search-query').value = '';
+
+    fetchCoordinates(city)
+      .then((coordinates) => {
+        fetchCurrentCityWeather(coordinates.lat, coordinates.lon, apiKey);
+        fetchFiveDayWeather(coordinates.lat, coordinates.lon, apiKey)
+      })
+
+});
+
+function capitalLetter(string){
```

Comment:
It is best practice to use function expressions instead of declarations and staying consistent in your code base